src/AddItems.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const updateGroceryItem = (event) => {
+    setGroceryItem(event.target.value);
+  };
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: document.querySelectorAll('input[name=frequency]:checked')[0]
+        .value,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+  };
+
+  return (
+    <div>
+      <form id="test-form" onSubmit={submitGroceryItem}>
+        <label htmlFor="item">Item name:</label>
+        <input
+          type="text"
+          id="item"
+          value={groceryItem}
+          onChange={updateGroceryItem}
+        />
+        <input type="radio" id="soon" value={7} name="frequency" />
+        <label htmlFor="soon">Soon (in the next 7 days)</label>
+        <input type="radio" id="kindOfSoon" value={14} name="frequency" />
+        <label htmlFor="kindOfSoon">Kind of soon (in the next 14 days)</label>
+        <input type="radio" id="notSoon" value={30} name="frequency" />
+        <label htmlFor="notSoon">Not soon (in the next 30 days)</label>
```

Comment:
Similar to the text input, it would be a good idea to incorporate an `onChange` event with `useState` to control these inputs. That way, you can just just use that value of that state instead of `document.querySelectorAll` above.